app/views/transactions/_form.html.slim

```diff
@@ -29,14 +29,21 @@
         collection: current_organization.bank_accounts.visible.positioned.grouped_by_currency(current_organization.default_currency),
         prompt: 'Bank account', group_method: :last, label: 'From',
         selected: t.object.bank_account_id
-
+  - elsif params[:invoice_id] && params[:customer_id] || f.object.invoice_id
+    = f.input :customer_name, as: :hidden
```

Comment:
@yunixon I think it will be better to not hide this field, but make is disabled, so we will be informed what customer will be set to transaction.